src/Goerli-DssSpell.t.sol

```diff
@@ -296,7 +328,7 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         assertEq(uint256(toc), 0);
     }
 
-    function testFailSpellIsCast_RWA008AT3_INTEGRATION_CURE() public {
+    function testFailSpellIsCast_RWA009AT1_INTEGRATION_CURE() public {
```

Comment:
Same as above. Since `cure()` cannot be called successfully, we can make it assert that it will always fail..